specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzetext-authoring.json

```diff
@@ -1104,30 +1531,72 @@
           "Successful Cancel Training Job": {
             "$ref": "./examples/analyzetext-authoring/SuccessfulCancelTrainingJob.json"
           }
-        },
-        "x-ms-long-running-operation": true
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/authoring/analyze-text/projects/global/deletion-jobs/{jobId}": {
```

Comment:
What creates a type of job that uses this for a status monitor? I don't see anything obvious in this swagger.

Reply:
That would be the delete project API. Abbas who was on my team has asked the board about this before. The issues he faced was that the get status for the delete project operation couldn't projects/{projectName}/jobs/{jobId} because as soon as the project is actually deleted, then the get status would return a 404. So the board recommendation was to use this for create project get status.

This is the same as what we have in our 2022-05-01 GA APIs by the way, the signature hasn't changed.